Earthfile

```diff
@@ -24,6 +24,18 @@ check-bash:
 
     DO ./earthly/bash+SHELLCHECK --src=.
 
+## -----------------------------------------------------------------------------
+##
+## Standard CI targets.
+##
+## These targets are discovered and executed automatically by CI.
+
+# check run all checks.
+check:
```

Comment:
Remove this check and the comment above

Reply:
we already have the wildcard implemented so no need for it.

Reply:
https://github.com/input-output-hk/catalyst-ci/actions/runs/7530410352/job/20496740717?pr=148

We can see that in `RUN` `check`, `check-spelling`, `check-markdown`, and `check-bash` are running.

Reply:
ohhh roger krub